.lagoon.yml

```diff
@@ -92,7 +104,19 @@ tasks:
           set -e
           drush en -y dpl_example_content
         service: cli
+    # @TODO How do we run this, even if something has gone wrong?
+    - run:
+        name: Setting Deployment status
+        command: |
+          # @TODO How do we set this to 'failure', if something has gone wrong?
+          DEPLOYMENT_STATUS="success";
```

Comment:
Good question. Perhaps this video helps? https://docs.lagoon.sh/resources/faq/#how-can-i-check-the-status-of-a-build

You could also [ask the Lagoon team on Discord](https://docs.lagoon.sh/community/discord/).